ci/jenkins/nginx.conf.tpl

```diff
@@ -0,0 +1,42 @@
+server {
+    server_name ci.ipfs.team;
+    access_log /var/log/nginx/access.log mtail;
+    listen 80;
+    listen [::]:80;
+
+    location / {
+        proxy_pass http://127.0.0.1:8090;
+        proxy_set_header Host \$host;
+    }
+
+    location /prometheus {
+        return 403;
+    }
+}
+
+server {
+    access_log /var/log/nginx/access.log mtail;
+    listen [$(var cjdns_ipv6)]:8090;
```

Comment:
Not sure if this is correct. Currently having issues with the :8090 being busy on the host so can't start the jenkins container, guessing this is the culprit of why.